src/main/java/frc/robot/constants/IntakeConstants.java

```diff
@@ -0,0 +1,7 @@
+package frc.robot.constants;
+
+public class IntakeConstants {
+	public static final int motorId = 5;
+	public static final int sensorId = 2;
+	public static final int maxMotorSpeed = 80;
```

Comment:
motor speed runs between -1 and 1. maxMotorSpeed should be a double, probably .80